pkgs/tools/networking/acme-client/default.nix

```diff
@@ -12,24 +15,27 @@ with stdenv.lib;
 
 stdenv.mkDerivation rec {
   pname = "acme-client";
-  version = "0.1.16";
+  version = "0.2.4";
 
-  src = fetchurl {
-    url = "https://kristaps.bsd.lv/acme-client/snapshots/acme-client-portable-${version}.tgz";
-    sha256 = "00q05b3b1dfnfp7sr1nbd212n0mqrycl3cr9lbs51m7ncaihbrz9";
+  src = fetchFromGitHub {
+    owner = "graywolf";
+    repo = "acme-client-portable";
+    rev = "v${version}";
+    sha256 = "1yq2lkrnjwjs0h9mijqysnjmr7kp4zcq1f4cxr9n1db7pw8446xb";
   };
 
-  buildInputs = [ libbsd libressl pkgconfig ]
+  buildInputs = [ autoconf automake bison libbsd libressl pkgconfig ]
     ++ optional stdenv.isDarwin apple_sdk.sdk;
 
   CFLAGS = "-DDEFAULT_CA_FILE='\"${defaultCaFile}\"'";
 
   preConfigure = ''
+    autoreconf --install
     export PREFIX="$out"
   '';
```

Comment:
The whole hook can become:
```nix
    makeFlags = [ "PREFIX=${placeholder "out"}" ];
```
and autoreconf is taken care of through nativeBuildInputs